packages/frontend/src/components/screens/MainScreen/MainScreen.tsx

```diff
@@ -536,7 +527,7 @@ function ChatNavButtons({
         // Core only allows placing calls in chats of type "single"
         // (but not e.g. in groups consisting of 2 members).
         // https://github.com/chatmail/core/blob/738dc5ce197f589131479801db2fbd0fb0964599/src/calls.rs#L147
-        chat.chatType === C.DC_CHAT_TYPE_SINGLE &&
+        chat.chatType === "Single" &&
```

Comment:
```suggestion
        chat.chatType === 'Single' &&
```

This will fix all lint warnings.